packit/constants.py

```diff
@@ -190,6 +190,11 @@
     "```\ngit fetch https://src.fedoraproject.org/forks/{user}/rpms/{package}.git "
     "refs/heads/*:refs/remotes/{user}/*\n"
     "git checkout {user}/{branch}\n```\n"
+    "\n\n"
+    "Before pushing builds/updates, please remember to:\n"
+    "- check that the new sources only contain permissible content\n"
+    "- check that the license of the new version has not changed\n"
+    "- check for any API/ABI and other changes that may break dependent packages\n"
```

Comment:
Sadly, it looks like Pagure is not able to render task lists..;(

Reply:
It's PR description, it renders markdown, or at least it tries :D

Reply:
I've tried to edit an existing PR and it didn't work:

![image](https://github.com/packit/packit/assets/20214043/fef763d3-e960-4c00-8a4b-05a43d6cdf7e)
![image](https://github.com/packit/packit/assets/20214043/0efbcea1-784a-4b83-bf75-dc01c02c0514)

Reply:
oh you mean the task lists, no those aren't supported, but the lists themselves are…

(task lists are a mess anyways… apart from unchecked and checked there are also other states that are not universally supported everywhere :D)